From 7c21fa97815ac19c77a01db82c2e30834155c1b6 Mon Sep 17 00:00:00 2001
From: Nils Philippsen <nils@redhat.com>
Date: Thu, 4 Aug 2011 12:10:04 +0200
Subject: [PATCH] patch: startup-warning

Squashed commit of the following:

commit 083d5fc4211421f900a2ee3ba9967be4ef9bb5eb
Author: Mikael Magnusson <mikachu@src.gnome.org>
Date:   Wed Mar 9 15:35:52 2011 +0100

    app: fix goption warning

    Using G_OPTION_FLAG_NO_ARG with G_OPTION_ARG_NONE is not a valid combination,
    glib 2.28.2 warns about it.

    (gimp:20379): GLib-WARNING **: goption.c:2132: ignoring no-arg, optional-arg or filename flags (8) on option "debug-handlers" of type 0
    (cherry picked from commit b81276ca83e6ec4c7df32c8f611f441d1dfc59ad)
---
 app/main.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/app/main.c b/app/main.c
index e1d5f02..34c9f74 100644
--- a/app/main.c
+++ b/app/main.c
@@ -240,7 +240,7 @@ static const GOptionEntry main_entries[] =
     N_("Debug in case of a crash (never|query|always)"), "<mode>"
   },
   {
-    "debug-handlers", 0, G_OPTION_FLAG_NO_ARG,
+    "debug-handlers", 0, 0,
     G_OPTION_ARG_NONE, &use_debug_handler,
     N_("Enable non-fatal debugging signal handlers"), NULL
   },
-- 
1.7.6

